btx/processing/peak_finder.py

```diff
@@ -15,7 +15,7 @@ class PeakFinder:
     https://confluence.slac.stanford.edu/display/PSDM/Hit+and+Peak+Finding+Algorithms
     """
     
-    def __init__(self, exp, run, det_type, outdir, tag='', mask=None, psana_mask=True, det_dist=None,
+    def __init__(self, exp, run, det_type, outdir, tag='', mask=None, psana_mask=True, clen=None,
```

Comment:
do we want to keep `clen` or use `camera_length` in `btx` and translate it to `clen` when it talks to psana/psgeom?

Reply:
Thanks for pointing that out! I actually didn't realize that's what it was an abbreviation for, but I'll change it to `camera_length` since that's more descriptive.